compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/ObserverbilitySymbolCollectorRunner.java

```diff
@@ -0,0 +1,44 @@
+package org.wso2.ballerinalang.compiler.semantics.analyzer;
```

Comment:
Missing Licence header.

Reply:
Added in the latest commit. I didn't have the copyright settings configured. I fixed it in the IDE.